_pages/branding.html

```diff
@@ -19,7 +19,7 @@ <h5>Colors</h5>
                     <p>#d12727</p>
                 </td>
                 <td class="color">
-                    <p>Black</p>
+                    <p>Grey</p>
```

Comment:
I still think of this as black in accordance with the school colors, but it's up to y'all

Reply:
It does look slightly grey, but as Erik said, it's supposed to be black to match school colors. If the choice is between calling dark grey black or changing the color we use as a team, I'd say it's easier to keep the color and name.